cylc/uiserver/resolvers.py

```diff
@@ -194,9 +197,15 @@ async def mutator(self, info, *m_args):
             return [{
                 'response': (False, 'No matching workflows')}]
         # Pass the request to the workflow GraphQL endpoints
-        req_str, variables, _, _ = info.context.get('graphql_params')
+        _, variables, _, _ = info.context.get('graphql_params')
+
+        # Create a modified request string,
+        # containing only the current mutation/field.
+        operation_ast = deepcopy(info.operation)
+        operation_ast.selection_set.selections = info.field_asts
+
         graphql_args = {
-            'request_string': req_str,
+            'request_string': print_ast(operation_ast),
             'variables': variables,
         }
```

Comment:
Won't this have to be done for `nodes_mutator()` below too? If so it's probably worth extracting this out into a helper function?

Reply:
(not the helper function bit.. but it's only 2 places, a small section, and I don't think it'll be used elsewhere)